CODEOWNERS

```diff
@@ -199,6 +199,9 @@
 # The `compass` chart
 /resources/compass/ @PK85 @aszecowka @crabtree @pkosiec @kfurgol @tgorgol
 
+# The `compass-runtime-agent` chart
+/resources/compass/ @akgalwas @janmedrek @Szymongib @franpog859 @Maladie
```

Comment:
I'm not sure if it should be a separate chart. Cause with one chart, we will can 
 use subcharts and overrides. We need to discuss that.